litezip/validate.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+import re
+
+
+__all__ = (
+    'is_valid_identifier',
+)
+
+
+VALID_ID_REGEX = re.compile("^(col|m)\d{5}$")
```

Comment:
this should be `^(col\d{5,}\d*|m\d{4,})$` The first module is `m0001` (but the first collection is `col10040` for slightly historical reasons) and the commas to allow for more, in the future.